pilot/pkg/security/authn/utils/utils_test.go

```diff
@@ -18,10 +18,24 @@ import (
 	"testing"
 
 	tls "github.com/envoyproxy/go-control-plane/envoy/extensions/transport_sockets/tls/v3"
+	"github.com/google/go-cmp/cmp"
+	"google.golang.org/protobuf/testing/protocmp"
 
 	meshconfig "istio.io/api/mesh/v1alpha1"
+	model "istio.io/istio/pilot/pkg/model"
+	"istio.io/istio/pilot/pkg/networking"
 )
 
+// SupportedCiphers for server side TLS configuration.
+var DefaultSupportedCiphers = []string{
```

Comment:
Why do we need this? Its only used for tests and we already have `utils.SupportedCiphers` defined

Reply:
yes, we can re-use. Modified the code accordingly. Thanks.